### ISSUE TITLE
Button hover effect missing

### ISSUE THREAD

**Issue by Nithya-Satish-ACAD** [Rashika-S/Web-Development#3]
The “Add to Cart” button lacks a hover/focus state, which improves interactivity and accessibility.

Suggested Fix:

```
.add-to-cart-button:hover {
  background-color: hsl(158, 42%, 18%);
  transition: 0.3s ease;
}

```
